assemble/Construct_Momentum_Element_DG.F90

```diff
@@ -0,0 +1,356 @@
+#include "fdebug.h"  
+#undef ASSERT
```

Comment:
The standard assert function uses the Fortran continuation operator, //, which the c preprocessor then sees as a c-style comment and strips from the the processed template file. There may be a way to prevent this at the command line but I couldn’t find a version which worked. 


> On 12 Jul 2016, at 17:18, Stephan Kramer <notifications@github.com> wrote:
> 
> In assemble/Construct_Momentum_Element_DG.F90:
> 
> > @@ -0,0 +1,356 @@
> > +#include "fdebug.h"  
> > +#undef ASSERT
> 
> Why are we redefining ASSERT here?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>